recipes-core/image/image.bb

```diff
@@ -0,0 +1,19 @@
+SUMMARY = "A small image just capable of allowing a device to boot."
```

Comment:
I'm not sure, but name `image.bb` looks very general. May be we may find something more specific or use append to something like core-image-minimal?